subcommands/devices/trigger.go

```diff
@@ -0,0 +1,139 @@
+package devices
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+	nanoid "github.com/matoous/go-nanoid/v2"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+)
+
+func init() {
+	triggerCmd := &cobra.Command{
+		Use:   "trigger",
+		Short: "Trigger remote actions on device",
+		Long: `
+*NOTE*: Requires devices running LmP version 97 or later.
```

Comment:
Long help usually starts with the same line as the short help (without a leading newline). They are not being concatenated.